sites/partners/src/components/applications/PaperApplicationForm/sections/FormDemographics.tsx

```diff
@@ -27,17 +27,32 @@ const FormDemographics = ({ formValues }: FormDemographicsProps) => {
   // Does a key exist in a root field or a sub array
   const isKeyIncluded = (raceKey: string) => {
     let keyExists = false
-    formValues?.race?.forEach((value) => {
-      if (value.includes(raceKey)) {
-        keyExists = true
-      }
-    })
+    const curValues = getValues()
```

Comment:
These additions look duplicated in the other file, could we move it to shared-helpers? We could also reduce duplication in this section and the one below w a pattern like:
obj = (if statement) ? object.entries ... : formValues.race ...
 obj.forEach ...